tests/test_json_model.py

```diff
@@ -290,6 +289,14 @@ def test_no_data_wrapper_deserialize(self):
         content = model.response(resp, content)
         self.assertEqual(content, {"data": "is good"})
 
+    def test_no_data_wrapper_deserialize_text_format(self):
+        model = JsonModel(data_wrapper=False)
+        resp = httplib2.Response({"status": "200"})
+        resp.reason = "OK"
+        content = 'column1,column2,column3\nstring1,1.2,string2'
```

Comment:
Please make this a named constant, and use it in the assertion below.

Reply:
OK